hibernate-reactive-core/src/main/java/org/hibernate/reactive/event/impl/DefaultReactiveMergeEventListener.java

```diff
@@ -211,7 +215,7 @@ else if ( original instanceof PersistentAttributeInterceptable ) {
 		final EventSource source = event.getSession();
 		final EntityPersister persister = source.getEntityPersister( event.getEntityName(), entity );
 
-		copyCache.put( entity, entity, true );  //before cascade!
+		copyCache.put( entity, entity, true );  //before cas‘cade!
```

Comment:
I think this is a typo

Reply:
@DavideD I already fixed it, I think you're looking at one commit instead of the whole diff.

Reply:
Oh wait, I see. I accidentally had a commit from #954 on this branch. But I already fixed it.

Reply:
Sure, no problem